### PR TITLE
Simple aggregation of flows

### PR DESCRIPTION
**Pull request by dbgrigsby**
Singledecode without FlowDecode for now

### CODE REVIEW

File: aggregate.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        for matching_flow in matching_flows:
+            merged_flow = merge_flows(merged_flow, matching_flow)
+        merged_flows.append(merged_flow)
+
+    return set(merged_flows) # Remove duplicate merged flows
+
+
+def singledecode(flow_info, hosts):
+    all_cpus_identified_flows = set()
+    for cpu in range(0, CPUS):
```
Comment (mtbentley) [inline aggregate.py]: can you just do `for cpu in flow_info.keys()`?

File: aggregate.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        identified_flows = set()
+        merged_flows = flow_info[str(cpu)]
+        usable_flows = dict((hash_value, csv_tuple_to_dict(flow)) for hash_value, flow in merged_flows.items())
```
Comment (mtbentley) [inline aggregate.py]: FYI you can do `{hash_value: csv_tuple_to_dict(flow) for hash_value, flow in merged_flows.items()}`

**Review by mtbentley — APPROVED** [review]
Only a couple comments, but feel free to ignore them

File: aggregate.py (hunk 1 above)
Comment (dbgrigsby) [reply aggregate.py]: yep

File: aggregate.py (hunk 2 above)
Comment (dbgrigsby) [reply aggregate.py]: 👍

**Pull request merged**